app/assets/javascripts/services/search.js

```diff
@@ -0,0 +1,23 @@
+angular.module("OpenHq").factory("Search", function($http) {
+  return {
+    find: function(term, opts) {
+      opts = opts || {};
+      _.defaults(opts, {
+        page: 1,
+        limit: 20
+      });
+
+      return $http({
+        method: "GET",
+        url: "/api/v1/search",
+        params: {
+          term: term,
+          page: opts.page,
+          limit: opts.limit
+        }
+      });
```

Comment:
you can chain a promise on the end of 

```js
.then(function(resp) {
  return resp.data;
});
```

That way all the calling code just directly gets the json data back rather than the full http response object.